addons/account_peppol/wizard/peppol_registration.py

```diff
@@ -193,14 +210,22 @@ def _branch_with_same_address(self):
     def _ensure_mandatory_fields(self):
         if self.use_parent_connection:
             return
+        if not self.company_id.country_code:
```

Comment:
```suggestion
        if not self.company_id.account_fiscal_country_id.code:
```
Also, I think you should use `selected_company_id` here